src/styles/index.scss

```
@@ -1,3 +1,17 @@
-body {
+@import './utils/variables';
+@import './blocks/card';
+@import './blocks/stars';
+
+html {
+  font-family: Roboto, sans-serif;
+  font-weight: 400;
+  line-height: 14px;
+  font-style: normal;
```

It's default values for `font-weight` and `font-style` so they are redundant
```suggestion
  line-height: 14px;
```